corehq/apps/receiverwrapper/util.py

```diff
@@ -219,19 +219,14 @@ def _submitted_by_demo_user(form_meta, domain):
 
 
 def _notify_ignored_form_submission(request, form_meta):
```

Comment:
@mkangia @shubham1g5 I'm switching this email to a `soft_assert` so that it's disabled by default on non-Dimagi environments.  It should still have all the same info, but the format might be a bit different.

Reply:
cool.